db/seeds.rb

```diff
@@ -51,3 +51,24 @@ def time_floor(time, seconds = 15.minutes)
     end
   end
 end
+
+amenity_list = [
+      "Coffee",
+      "Projector",
+      "Windows",
+      "Internal AC",
+      "Speakers",
+      "Video camera",
+      "Microphone",
+      "White board"
+]
+8.times do
```

Comment:
try changing line 65 to:
```
amenity_list.each do |amenity_name|
  Amenity.create(name: amenity_name)
end
```
This will make the code more flexible because you can add more amenities in the amenity_list array without having to also update the amount of times iterating through the do block.